package/features/environment.py

```diff
@@ -63,20 +90,7 @@ def _setup_context_with_venv(context, venv_dir):
     context.env["PIP_DISABLE_PIP_VERSION_CHECK"] = "1"
 
     call(
-        [
-            context.python,
-            "-m",
-            "pip",
-            "install",
-            "-U",
-            "pip>=21.2",
-            "setuptools>=38.0",
-            "cookiecutter>=1.7.2",
-            "wheel",
-            "botocore",
-            "PyYAML>=4.2, <6.0",
-            "click<9.0",
-        ],
+        [context.python, "-m", "pip", "install", "-U", "pip>=21.2", "setuptools>=38.0"],
```

Comment:
are the removed requirements no longer needed ?

Reply:
yes, they were there because of kedro 16 i think. we stopped support for kedro 16 but had this bit of chore leftover to do